String Searching/boyer_moore.cpp

```diff
@@ -0,0 +1,113 @@
+#include <bits/stdc++.h>
+#define endl '\n'
```

Comment:
You don't need define this as präprocessor directive. Its contain in C++ (namespace std)